review.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTPYE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+  </head>
+  <body>
+    <main id="main">
+      <div id="title">
+        <h1>
+          Dr. Norman Borlaug
+        </h1>
+        <h4>
+          The man who saved a billion lives
+        </h4>
+      </div>
+
+      <div id="img-div"> 
+        <img id="image" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg" >
```

Comment:
You need to add an alt attribute whenever you use a <img> tag. The alt attribute provides alternative information for an image if a user for some reason cannot view it (because of a slow connection, an error in the src attribute, or if the user uses a screen reader). 👓